docs/services/aws-services-guide.md

```diff
@@ -34,154 +29,25 @@ local machine.
 ## Preparation
```

Comment:
^ As best as I can tell `jq` and the AWS CLI are no longer prereqs for this guide. 🎉